src/components/homepage/__test__/homepage.test.js

```diff
@@ -0,0 +1,36 @@
+import "@testing-library/jest-dom/extend-expect";
+import { cleanup, fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import Cookies from 'universal-cookie';
+import Homepage from './../homepage';
+
+afterEach(cleanup);
+
+describe('Homepage content', () => {
+    test('renders homepage correctly', () => {
+        render(<Homepage></Homepage>);
+        expect(screen.getByRole('button')).toBeInTheDocument();
+    });
+  });
+
+describe('Button event 1', () => {
+    test('Show alert message', () => {
+        window.alert = jest.fn();
+        render(<Homepage></Homepage>);
+        const node = screen.getByText('Logout');
+        fireEvent.click(node);
+        expect(window.alert).toBeCalledWith('Already logged out');
+    });
+  });
+  
+describe('Button event 2', () => {
```

Comment:
I have no idea what this test does by reading the describe statement, can you describe the scenario properly please?